### DISCUSSION TITLE
[List] How to add a scrollbar to a list and how to update the scrollbar when content changes

### DISCUSSION THREAD

**Discussion by JobberRT** [fomantic/Fomantic-UI#2320] [Elements (button, image, label, loader, ...)] [answered]
Hi, this is a simply question but i can't find a solution to it. I tried adding a ```scroll``` class to it but that doesn't work,  I even tried [SimpleBar](https://github.com/Grsmto/simplebar) but turns out that it will overflow out of the div when ```item``` is too many.

Is FomanticUI support a way of giving ```List``` a scrollbar? Is there any way to update the scrollbar when content is changed?

Example:  [JSFiddler](https://jsfiddle.net/76gte2z3/5/)

**Comment by lubber-de** [comment, marked as answer]
Add `overflow-y: scroll;` Scrollbar changes automatically when items are added
See https://jsfiddle.net/lubber/pqw7dh56/

**Reply by JobberRT** [reply]
Ohh~~Hahaha, I'm overthinking this. Thought Fomantic UI has some kinds of implement, turns out that just simple CSS will work hhahaha
